app/cdap/components/GridTable/index.tsx

```diff
@@ -57,6 +61,16 @@ export default function GridTable() {
     },
   ]);
 
+  const { dataprep } = DataPrepStore.getState();
+  const [isFirstWrangle, setIsFirstWrangle] = useState<boolean>(false);
+  const [connectorType, setConnectorType] = useState<string>(null);
```

Comment:
the interface migjt be string | null
plz check if this is throwing error

Reply:
Done